chartfx-math/src/test/java/de/gsi/math/spectra/ApodizationTests.java

```diff
@@ -0,0 +1,136 @@
+package de.gsi.math.spectra;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import de.gsi.math.spectra.Apodization.ApodizationArrayDescription;
+
+/**
+ * Test the apodization class
+ * 
+ * @author Alexander Krimm
+ */
+class ApodizationTests {
+    private static final double[] RECTANGULAR8 = new double[] { 1, 1, 1, 1, 1, 1, 1, 1 };
+    private static final double[] BLACKMAN8 = new double[] { -1.3877787807814457E-17, 0.09045342435412804,
+            0.45918295754596355, 0.9203636180999081, 0.9203636180999083, 0.45918295754596383, 0.09045342435412812,
+            -1.3877787807814457E-17 };
+    private static final double[] BLACKMANHARRIS8 = new double[] { 6.0000000000001025E-5, 0.03339172347815117,
+            0.332833504298565, 0.8893697722232837, 0.8893697722232838, 0.3328335042985652, 0.03339172347815122,
+            6.0000000000001025E-5 };
+    private static final double[] BLACKMANNUTTAL8 = new double[] { 3.628000000000381E-4, 0.03777576895352025,
+            0.34272761996881945, 0.8918518610776603, 0.8918518610776603, 0.3427276199688196, 0.0377757689535203,
+            3.628000000000381E-4 };
+    private static final double[] EXPONENTIAL8 = new double[] { 1.0, 1.0425469051899914, 1.086904049521229,
+            1.1331484530668263, 1.1813604128656459, 1.2316236423470497, 1.2840254166877414, 1.338656724353094 };
+    private static final double[] FLATTOP8 = new double[] { 0.004000000000000087, -0.16964240541774014,
+            0.04525319347985671, 3.622389211937882, 3.6223892119378833, 0.04525319347985735, -0.16964240541774012,
+            0.004000000000000087 };
+    private static final double[] HAMMING8 = new double[] { 0.07671999999999995, 0.2119312255330421, 0.53836,
+            0.8647887744669578, 1.0, 0.8647887744669578, 0.5383600000000001, 0.21193122553304222 };
+    private static final double[] HANN8 = new double[] { 0.0, 0.1882550990706332, 0.6112604669781572,
+            0.9504844339512095, 0.9504844339512095, 0.6112604669781573, 0.1882550990706333, 0.0 };
+    private static final double[] HANNEXP8 = new double[] { 0.0, 0.6112604669781572, 0.9504844339512096,
+            0.18825509907063334, 0.18825509907063315, 0.9504844339512096, 0.6112604669781574, 5.99903913064743E-32 };
+    private static final double[] NUTTAL8 = new double[] { -2.42861286636753E-17, 0.031142736797915613,
+            0.3264168059086425, 0.8876284572934416, 0.8876284572934416, 0.32641680590864275, 0.031142736797915654,
+            -2.42861286636753E-17 };
+
+    @Test()
+    public void testName() {
+        assertEquals("rectangular", Apodization.Rectangular.getName());
+    }
+    
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5084368)